stripper/ze_tyranny_v5_2k3.cfg

```diff
@@ -182,3 +182,60 @@ modify:
 		"OnUser1" "gravity_makerForceSpawn0.02-1"
 	}
 }
+
+;nerf ridiculous lightning damage it can be a issue with the lvl5 teleport before the Balrog boss fight
+modify:
+{
+	match:
+	{
+		"targetname" "lightning_hurt"
+	}
+	replace:
+	{
+		"damage" "150"
```

Comment:
Nerf seems a little too big, keep in mind this value is always double what it actually is ingame. So 2000 = 1000 (original value), 150 = 75 etc. Maybe 500 = 250 would be better?

Reply:
Done, in Castlevania 64 EW Config, you have wrong hammerid for the Ammo Crate Item.